diagnostics/collect-wsl-logs.ps1

```diff
@@ -14,7 +13,12 @@ mkdir -p $folder
 if ($LogProfile -eq $null)
 {
     $LogProfile = "$folder/wsl.wprp"
-    Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/microsoft/WSL/master/diagnostics/wsl.wprp" -OutFile $LogProfile 
+    try {
```

Comment:
I'm not a PS expert: What's the reason for this ? Does this give a better error ?

Reply:
Ok. Sounds good then